bin/mail/2016-03-21-teacher-recruitment/send-hoc2015-organizers-real-mails

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+# 157303 contacts
+../send-to-mailing-list --start      0 --limit 100000 3-17-recruit-hoc-code-studio ./hoc2015-organizers-deduped.csv
+../send-to-mailing-list --start 100000 --limit 100000 3-17-recruit-hoc-code-studio ./hoc2015-organizers-deduped.csv
```

Comment:
Good suggestion. I'm not actually sure why it does batches anyway. This is legacy. I increased the default to 100k some time ago (I think it was 20k before that) but we probably can get rid of it entirely.